narwhals/_expression_parsing.py

```diff
@@ -250,7 +250,8 @@ def is_simple_aggregation(expr: CompliantExpr[Any, Any]) -> bool:
 
         - nw.col('a').filter(nw.col('b')>nw.col('c')).max()
 
-    because then, we can use a fastpath in pandas.
+    Elementary ones are the only ones supported properly in
```

Comment:
```suggestion
    Elementary expressions are the only ones supported properly in
```
is this what you meant? :)